src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNSettings.java

```diff
@@ -83,6 +85,7 @@
     /**
      * Default setting values
      */
+    public static final String INDEX_KNN_DEFAULT_ENGINE = "Faiss"; // nmslib, faiss
```

Comment:
I think we should keep nmslib as the default engine.

Reply:
Yes, you are right. this is just for integration test can cover the `faiss` engine code.